src/utils.py

```
@@ -7,6 +7,17 @@
 from datetime import datetime, timedelta
 from src import *
 
+from google.cloud import secretmanager
+
+# Access the GitHub token from Google Secret Manager
+def access_secret_version(project_id, secret_id, version_id="latest"):
```

Ruff (0.8.2)

Missing return type annotation for public function `access_secret_version`

(ANN201)